scripts/github_issue_retriever.py

```diff
@@ -46,6 +86,20 @@ def retrieve_issues(
             logger.info("Retrieving all issues since last_modified is not available")
             github.download_issues(owner, repo, state, retrieve_events)
 
+            if retrieve_private:
+                logger.info(
+                    "Replacing contents of auto closed public issues with private issues content"
+                )
+
+                all_issues = list(github.get_issues())
+                _, updated_issues, updated_ids = replace_with_private(all_issues)
```

Comment:
If you apply the nit suggested above, this becomes `updated_issues, updated_ids = replace_with_private(data)`